css/index.css

```diff
@@ -51,42 +51,92 @@ table {
     box-sizing: border-box;
 }
 
+html {
+    font-size: 62.5%;
+}
 html, body {
     height: 100%;
     font-family: 'Roboto', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
+    font-size: 1.8rem;
+    margin-bottom: 1.5%;
     font-family: 'Rubik', sans-serif;
 }
 
 p {
     line-height: 1.4;
 }
-
+/* 1000px */
 .container {
     width: 800px;
+    max-width: 100%;
     margin: 0 auto;
+    font-size: 1.5rem;
+}
+
+header {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 1.8% 0;
 }
 
+
+header img {
+    margin-right: 5%;
+}
+header nav {
```

Comment:
You may want to take a look at the google inspect tool if you are using chrome. The nav and the header image are not quite aligned. This can be fixed with adding some margins in the header img portion of the css!